docs/getting_started/installation.rst

```diff
@@ -4,217 +4,145 @@
-- `pyet <https://github.com/pyet-org/pyet>`_ Provides processing functionalities of several methods to calculate evapotranspriation. Available in both conda-forge and pypi.
-- `gdal <https://gdal.org/en/stable/>`_ Provides many drivers and GIS transformations.  Only available through conda-forge.
+- `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_
+- `Miniforge (Mambaforge) <https://conda-forge.org/docs/>`_
+- `uv <https://docs.astral.sh/uv/>`_
 
-Since some dependencies are only available through conda-forge and some only through pypi, you will need a packange manager that can handle both.
+Both conda variants come preconfigured with the **conda-forge** channel, which provides free and open packages used by HydroMT.
 
-Installation
-============
+Installing HydroMT-Wflow
+========================
 
-There are various ways in which one can install hydrom_wflow in a python environment, the main being:
+HydroMT-Wflow is available from both **PyPI** and **conda-forge**.
+The simplest and most flexible approach is to install it using **pip** inside a new environment.
 
-- :ref:`Pixi <user_install_pixi>`
-- :ref:`Miniforge/conda <user_install_conda>`
+Installation using uv / pip
+----------------------------
 
-All of these options will be discussed below but we recommend using pixi, as it is the fastest and most
-robust (and reproducible) option.
-
-.. _user_install_pixi:
-
-Installation using Pixi
------------------------
-
-.. Tip::
-
-    This is our recommended way of installing HydroMT-Wflow!
-
-
-If you do not have a ``pyproject.toml`` yet you can make one by executing the command:
+We recommend creating a clean environment to avoid dependency conflicts. For example:
 
 .. code-block:: console
 
-    $ pixi init --format pyproject myproject
+    $ conda create -n hydromt-wflow uv python=3.11
+    $ conda activate hydromt-wflow
+    $ uv pip install hydromt_wflow
 
+This will install HydroMT-Wflow along with HydroMT core and all required dependencies.
 
-A new folder with a ``pyproject.toml`` called ``myproject`` will be created for you. After this, you can
-navigate to this new directory and add ``hydromt_wflow`` as a dependency:
+Installation using uv
+---------------------
 
-.. code-block:: console
+`uv <https://docs.astral.sh/uv/>`_ is a fast drop-in replacement for pip and virtualenv.
+It creates lightweight virtual environments and installs dependencies using a lockfile-based resolver for reproducibility.
 
-    $ pixi add hydromt_wflow[extra] --pypi
-
-Pixi will then add it as a dependency to the project. The ``[extra]`` instruct pixi to also
-include some optional dependencies.
-
-If you don't want these extra dependencies (or need specific versions, or only want some)
-
-You can also add it like so:
+To add HydroMT-Wflow to an existing uv environment:
```

Comment:
How do you create one? We were told that install guide was too complex because of too many options. Shall we skip this paragraph to keep it simple?